docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  db:
+    image: postgres
+    volumes:
+      - pg-data:/var/lib/postgresql/data
+  redis:
+    image: redis
+    volumes:
+      - redis-data:/data
+  web:
+    build: .
+    command: bundle exec rails s
+    depends_on:
+      - db
+      - redis
+    environment:
+      - RAILS_ENV=production
+      - DATABASE_URL=postgresql://postgres:@db:5432/intercity
+      - SECRET_KEY_BASE=a
+      - RAILS_SERVE_STATIC_FILES=true
+      - REDIS_URL=redis://redis:6379
+      - HOSTNAME=intercity.example.com
+      - RAILS_LOG_TO_STDOUT=true
+  sidekiq:
+    image: intercitynext_web
+    command: bundle exec sidekiq -q default -q mailers -q health_checks
+    environment:
+      - RAILS_ENV=production
+      - DATABASE_URL=postgresql://postgres:@db:5432/intercity
+      - REDIS_URL=redis://redis:6379
+      - SECRET_KEY_BASE=a
+      - RAILS_SERVE_STATIC_FILES=true
+      - HOSTNAME=intercity.example.com
+      - RAILS_LOG_TO_STDOUT=true
+    depends_on:
+      - db
+      - redis
+volumes:
+  pg-data:
```

Comment:
Is this OK to be empty?

Reply:
@jvanbaarsen Yeah, that indicates that it should just create a default Docker volume with no additional options.

Reply:
:+1: